src/main/js/utils/generate.js

```diff
@@ -71,34 +71,21 @@ const generateSchedules = (values, loadQuery = '') => (dispatch, getState) => {
           const instructor1 = getInstructorLastName(course1.instructor); // Backend only holds instructors' last name
           const instructor2 = getInstructorLastName(course2.instructor);
 
-          /** Can't sort if the backend doesn't hold data for the entire course */
           if (!gradeMap[name]) return 0;
 
-          /** Use mapping to determine sort order */
           if (!gradeMap[name][instructor1] && !gradeMap[name][instructor2]) return 0;
-          else if (!gradeMap[name][instructor1]) return 1;
-          else if (!gradeMap[name][instructor2]) return -1;
-          else {
-            return gradeMap[name][instructor2] - gradeMap[name][instructor1];
-          }
+          if (!gradeMap[name][instructor1]) return 1;
+          if (!gradeMap[name][instructor2]) return -1;
+          return gradeMap[name][instructor2] - gradeMap[name][instructor1];
         }));
       })
+      .catch(error => error)
```

Comment:
Catching and returning the error does nothing, this is where I wanted you to make use of passing the error into the action you just enabled to take errors.

Now that I think about it there no need for the .catch on the `getGPAMap` function anymore if we have this .catch here.

Reply:
Fixed and Removed .catch in getGPAMap()